Backendless/Persistence/DataQueryBuilder.cs

```diff
@@ -138,13 +139,19 @@ public DataQueryBuilder SetGroupBy( List<String> groupBy )
       return this;
     }
 
-    public DataQueryBuilder AddGroupdBy( List<String> groupdBy )
+    public DataQueryBuilder AddGroupBy( List<String> groupBy )
     {
-      this.groupBy = this.groupBy != null ? this.groupBy : new List<String>();
+      this.groupBy = this.groupBy ?? new List<String>();
       this.groupBy.AddRange( groupBy );
       return this;
     }
 
+    public DataQueryBuilder AddGroupBy ( String groupBy )
+    {
+         this.groupBy = new List<string> { groupBy };

```

Comment:
this is VERY wrong. You basically completely removed existing value in this.groupBy and assigned a new list to it.